Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -104,43 +104,48 @@ public async Task SetUp(CancellationToken cancellationToken = default)
             await RefreshExistingBuckets(cancellationToken);
         }
 
-        public void SelectBucket(string name)
+        public bool SelectBucket(string name)
         {
             if (BucketName == name)
             {
-                return;
+                return false;
             }
 
             BucketName = name;
 
             // Reset if we select a new valid bucket
             _status.IsDisplayed &= string.IsNullOrEmpty(BucketName);
+            return true;
         }
 
-        public void CreateBucket()
+        public Response CreateBucket()
```

Comment:
Same question as above. Is this PR potentially missing some changes where this is being used?